<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xMessageBox</class>
 <widget class="QDialog" name="xMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">模态消息框</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 28px &quot;微软雅黑&quot;;</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutDialog">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frameTitle
{
	background-color: transparent;
	border-image: url(:/img/widget/DialogTitle.png) 0 12 0 12;
	border-left: 12px transparent;
	border-right: 12px transparent;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutTitle">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelTitle
{
	color: white;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string notr="true">模态消息框</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameMainArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="IsMainArea" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutFrame">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="labelImage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelMessage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutBtns">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignRight|Qt::AlignVCenter">
         <widget class="xPixmapButton" name="BtnF1">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="imageNormal">
           <pixmap resource="xPad.qrc">:/img/widget/BtnBoxNormal.png</pixmap>
          </property>
          <property name="imagePress">
           <pixmap resource="xPad.qrc">:/img/widget/BtnBoxPress.png</pixmap>
          </property>
          <property name="borderTop">
           <number>8</number>
          </property>
          <property name="borderRight">
           <number>8</number>
          </property>
          <property name="borderBottom">
           <number>8</number>
          </property>
          <property name="borderLeft">
           <number>8</number>
          </property>
          <property name="contentAlignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight|Qt::AlignVCenter">
         <widget class="xPixmapButton" name="BtnF2">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="imageNormal">
           <pixmap resource="xPad.qrc">:/img/widget/BtnBoxNormal.png</pixmap>
          </property>
          <property name="imagePress">
           <pixmap resource="xPad.qrc">:/img/widget/BtnBoxPress.png</pixmap>
          </property>
          <property name="borderTop">
           <number>8</number>
          </property>
          <property name="borderRight">
           <number>8</number>
          </property>
          <property name="borderBottom">
           <number>8</number>
          </property>
          <property name="borderLeft">
           <number>8</number>
          </property>
          <property name="contentAlignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xPixmapButton" name="BtnF3">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="imageNormal">
           <pixmap resource="xPad.qrc">:/img/widget/BtnBoxNormal.png</pixmap>
          </property>
          <property name="imagePress">
           <pixmap resource="xPad.qrc">:/img/widget/BtnBoxPress.png</pixmap>
          </property>
          <property name="borderTop">
           <number>8</number>
          </property>
          <property name="borderRight">
           <number>8</number>
          </property>
          <property name="borderBottom">
           <number>8</number>
          </property>
          <property name="borderLeft">
           <number>8</number>
          </property>
          <property name="contentAlignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xPixmapButton</class>
   <extends>QWidget</extends>
   <header>xpixmapbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xPad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnF1</sender>
   <signal>clicked()</signal>
   <receiver>xMessageBox</receiver>
   <slot>BtnF1Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnF2</sender>
   <signal>clicked()</signal>
   <receiver>xMessageBox</receiver>
   <slot>BtnF2Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnF3</sender>
   <signal>clicked()</signal>
   <receiver>xMessageBox</receiver>
   <slot>BtnF3Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BtnF1Clicked()</slot>
  <slot>BtnF2Clicked()</slot>
  <slot>BtnF3Clicked()</slot>
 </slots>
</ui>
